dashboard/views/_positions/_ec/_secretary/_brother_management.py

```diff
@@ -29,32 +31,23 @@ def secretary_brother_view(request, brother_id):
     return render(request, "brother-view.html", context)
 
 
-@verify_position(['Secretary', 'Vice President', 'President', 'Adviser'])
+@verify_position(['secretary', 'vice-president', 'president', 'adviser'])
 def secretary_brother_add(request):
     """ Renders the Secretary way of viewing a brother """
     form = BrotherForm(request.POST or None)
 
     if request.method == 'POST':
         if form.is_valid():
             instance = form.cleaned_data
-            if instance['password'] == instance['password2']:
-                user = User.objects.create_user(instance['case_ID'], instance['case_ID'] + "@case.edu",
-                                                instance['password'])
-                user.last_name = instance['last_name']
-                user.save()
-
-                brother = form.save(commit=False)
-                brother.user = user
-                brother.save()
-
-                return HttpResponseRedirect(reverse('dashboard:secretary_brother_list'))
-            else:
-                context = {
-                    'error_message': "Please make sure your passwords match",
-                    'title': 'Add New Brother',
-                    'form': form,
-                }
-                return render(request, 'model-add.html', context)
+            user = User.objects.create_user(instance['case_ID'], instance['case_ID'] + "@case.edu",
+                                            instance['password'])
+            user.last_name = instance['last_name']
+            user.save()
+
+            brother = form.save(commit=False)
+            brother.user = user
+            brother.save()
+            return HttpResponseRedirect(reverse('dashboard:secretary_brother_list'))
```

Comment:
dont you have this same code in the marshal add candidate view with maybe a few minor changes?

Reply:
probably could make a `create_brother` helper function or something like that

Reply:
These two views are actually exactly the same. I'll just have both use the same view and make it more generic

Reply:
Actually, looking at it, both the secretary and the marshal had a LOT of the same views, so I combined all of the extremely similar ones